webpack.config.js

```diff
@@ -20,21 +20,36 @@ var path = require('path'),
             publicPath: siteRoot + publicStaticRoot,
             filename: 'ui-toolkit-doc-factory.js'
         },
-        modulesDirectories: ['node_modules'],
         resolve: {
             alias: {
                 afontgarde: 'edx-pattern-library/js/afontgarde',
                 modernizr: 'edx-pattern-library/js/modernizr-custom',
                 'edx-pattern-library': patternLibraryPath,
                 'edx-ui-toolkit': path.resolve(__dirname, 'src'),
                 doc: path.resolve(__dirname, 'doc/static')
-            }
+            },
+            modules: ['node_modules']
         },
         module: {
-            loaders: [
+            rules: [
                 {
                     test: /\.scss$/,
-                    loader: ExtractTextPlugin.extract('style', 'css!sass')
+                    loader: ExtractTextPlugin.extract({
+                        fallback: 'style-loader',
+                        use: [{
+                            loader: 'css-loader'
+                        }, {
+                            loader: 'sass-loader',
+                            options: {
+                                includePaths: [
+                                    path.resolve(__dirname, './node_modules'),
+                                    path.resolve(__dirname, './node_modules/edx-pattern-library/node_modules')
```

Comment:
AFAIK now that we're on NPM 3 there won't be any nested deps in here so line 45 would be sufficient to find all the includePaths